integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/functional-exceptions/FunctionalExceptionsEncryptionKeys.feature

```diff
@@ -12,12 +12,11 @@ Feature: SmartMetering functional exceptions regarding encryption keys
       | Channel                        |                                                                1 |
       | MbusIdentificationNumber       |                                                         24000000 |
       | MbusManufacturerIdentification | LGB                                                              |
-      | MbusUserKey                    | 17ec0e5f6a3314df6239cf9f1b902cbfc9f39e82c57a40ffd8a3e552cc720c92 |
+      | MbusUserKey                    | MBUS_USER_KEYPAIR |
```

Comment:
I think there are two schools - one making explicite steps and one making explicite by parameters. I think it's out of scope of this PR to make changes to the way it works now. Replacement of the hex coded key by a key name was intiated from the use of the same step in another scenario

Reply:
I don't I am really into any of the schools of step descriptions versus step data. I think information should be at the location where it makes the Cucumber feature most effective for all stake holders, this could be the step text, among the step data or in the code behind for more technical implementation details.